language/tools/test_generation/src/config.rs

```diff
@@ -5,35 +5,43 @@ use structopt::StructOpt;
 
 /// This defines how tolerant the generator will be about deviating from
 /// the starting stack height.
+/// Default is `0.9`
```

Comment:
Default implies that this could change, yet it is a constant. I'm confused.

Reply:
It can be changed before building the tool, but it won't change while the tool is running.